modules/connatixBidAdapter.js

```diff
@@ -0,0 +1,180 @@
+import {
+  deepAccess,
+  isFn,
+  logError,
+  isArray
+} from '../src/utils.js';
+
+import {
+  registerBidder
+} from '../src/adapters/bidderFactory.js';
+import {
+  BANNER,
+} from '../src/mediaTypes.js';
+
+const BIDDER_CODE = 'connatix';
+const AD_URL = 'https://capi.connatix.com/rtb/hba';
+const DEFAULT_MAX_TTL = '3600';
+const DEFAULT_CURRENCY = 'USD';
+
+/*
+   * Get the bid floor value from the bid object, either using the getFloor function or by accessing the 'params.bidfloor' property.
+   * If the bid floor cannot be determined, return 0 as a fallback value.
+   */
+export function getBidFloor(bid) {
+  if (!isFn(bid.getFloor)) {
+    return deepAccess(bid, 'params.bidfloor', 0);
+  }
+
+  try {
+    const bidFloor = bid.getFloor({
+      currency: DEFAULT_CURRENCY,
+      mediaType: '*',
+      size: '*',
+    });
+    return bidFloor.floor;
+  } catch (err) {
+    logError(err);
+    return 0;
+  }
+}
+
+/*
+   * Wrap the provided bid, playerId, customerId, and scriptId in an HTML string
+   * that includes the Connatix player script and related data.
+   * This HTML string will be used as the ad content.
+   */
+export function wrapAd(lineItems, requestId, playerId, customerId) {
+  var settings = {
+    advertising: {
+      standaloneLineItems: lineItems
+    }
+  };
+  var scriptSrc = `//cd.connatix.com/connatix.player.js?cid=${customerId}`;
```

Comment:
there are no protocol relative urls allowed in the project

Reply:
@patmmccann I updated and removed this part. We no longer needed it.